assets/sass/components/setup/_googlesitekit-setup.scss

```diff
@@ -230,7 +230,10 @@
 
 .googlesitekit-setup__wrapper--adsense-connect {
 
-	.googlesitekit-setup-module__footer-text {
-		font-size: 12px;
+	p {
+
+		&.googlesitekit-setup-module__footer-text {
```

Comment:
See above, I think these elements might probably be better to use a div instead.

Alternatively, adding the wrapping `.googlesitekit-plugin` would solve the specificity problem. Let's avoid making `p`+class-specific styles where possible.

Reply:
👍